addon/components/o-s-s/array-input.hbs

```diff
@@ -1,11 +1,7 @@
 <div class="array-input-container fx-row fx-xalign-center" ...attributes>
   {{#each this.items as |item index|}}
-    <div class="input-array-tag fx-row fx-xalign-center">
-      {{item}}
-      <i class="fa fa-remove margin-left-xxx-sm text-color-blue-gray" role="button"
-         {{on "click" (fn this.removeItem index)}}></i>
-    </div>
+    <OSS::Chip @label={{item}} @onRemove={{fn this.removeItem index}} />
   {{/each}}
-  <Input @value={{this.currentValue}} @placeholder={{@placeholder}} {{on "keydown" this.keyListener}}
-         autocomplete="off" {{on "blur" this.validateTagOnClickOutside}} />
+  <Input @value={{this.currentValue}} @placeholder={{@placeholder}} {{on "keydown" this.keyListener}} autocomplete="off"
```

Comment:
can you move the `autocomplete="off"` just after the placeholder arg ? we can see all the `on` modifiers together